src/control/server/ctl_storage_rpc.go

```diff
@@ -151,6 +159,112 @@ func (c *ControlService) scanScm(ctx context.Context, req *ctlpb.ScanScmReq) (*c
 	return newScanScmResp(c.getScmUsage(ssr))
 }
 
+// Return the number of target managing a given NVME controleur
+func (c *ControlService) getTargetCount(pciAddr string) (int, error) {
+	for _, engineCfg := range c.srvCfg.Engines {
+		targetCount := engineCfg.TargetCount
+		for _, tierCfg := range engineCfg.Storage.Tiers {
+			if tierCfg.IsBdev() {
```

Comment:
nit: tend not to want to nest too deep, instead, break early. here use 'if !tierCfg.IsBdev() {continue' to break the loop and then you can de-indent the rest of the block. makes it nicer to read